plugin/path_config.go

```diff
@@ -85,18 +81,13 @@ func (backend *ArtifactoryBackend) pathConfigWrite(ctx context.Context, req *log
 	}
 
 	if baseURL, ok := data.GetOk("base_url"); ok {
-		url := appendTrailingSlash(baseURL.(string))
-		cfg.BaseURL = url
+		cfg.BaseURL = baseURL.(string)
```

Comment:
Do you know what was the reason for `appendTrailingSlash` being used here?

Reply:
Artifactory will fail to start up if the given URL does not have a trailing slash, but it is handled in a different function now.